docs/how-to-run.adoc

```diff
@@ -2,24 +2,131 @@
 
-== Installing Prerequisite Software
+1. Using one or more simulated Sled Agents.  There are no real VMs.  All components listen on localhost and talk to each other directly.  The automated tests in this repo generally use this kind of deployment.  This is the only mode that's supported on non-illumos systems (i.e., Linux and MacOS).
+2. Using one or more real Sled Agents (on separate physical machines) using SoftNPU-based Boundary Services.  You can provision real VMs this way.  Intra-"rack" networking and boundary Services (i.e., external connectivity) are fully functional using SoftNPU, a software-based simulation of the Tofino device that's normally responsible for these functions.
++
+Within this mode, you can run on real Oxide hardware ("Gimlets") or ordinary server PCs.
+
+These are historically called "simulated" vs. "non-simulated", though they both involve simulation of some parts of a real Oxide system.
+
+Here's a summary of the tradeoffs:
+
+[cols="1h,1,1", options="header"]
+|===
+|
+|"Simulated" deployment
+|"Non-simulated" deployment
+
+|Works on
+|illumos, Linux, MacOS
+|illumos only
+
+|Nexus
+|Real implementation
+|Real implementation
+
+|CockroachDB
+|Real implementation, 1-node cluster
+|Real implementation, multi-node cluster (coming soon)
+
+|Internal/External DNS
+|Real implementation
+|Real implementation
+
+|Crucible Pantry
+|Real implementation
+|Real implementation
+
+|Sled Agent
+|Simulated, multiple okay
+|Real implementation, requires separate machines for each
+
+|Propolis
+|Missing (VMs are faked-up by simulated Sled Agent)
+|Real implementation
+
+|Dendrite
+|Stub implementation
+|Simulated implementation
+
+|Management Gateway Service (MGS)
+|Missing
+|Real implementation
+
+|Internal ("rack") networking
+|Localhost
+|Real implementation on-sled, simulated implementation across sleds (SoftNPU)
+
+|Boundary services (external connectivity)
+|Localhost
+|Simulated implementation (SoftNPU)
+
+|Wicket
+|Missing
+|Not used
+
+|Management network
+|Missing
+|Not used
+
+|===
 
-A major prerequisite is to have a machine already running Helios. An easy way to
-do this is by using a https://github.com/oxidecomputer/helios-engvm[Helios VM].
-ISOs are also available for download https://pkg.oxide.computer/install[here].
+To run the simulated control plane, refer to the guide on xref:how-to-run-simulated.adoc[running simulated Omicron].  The rest of this document describes how to run a single-Sled Omicron with a real Sled Agent and SoftNPU-based Boundary Services.
 
-The steps below will install several executables that will need to be in your
-`PATH`.  You can set that up first using:
+== Prerequisites
+
+=== Bare-metal operating system
+
+This guide assumes that you have a system running https://github.com/oxidecomputer/helios[Helios] on a bare-metal system.footnote:[You can in principle use a VM, but you wouldn't be able to provision Instances because nested virtualization is not supported.]
+
+=== External networking
```

Comment:
@rcgoodfellow @luqmana @internet-diglett (who did I miss?) I'd be interested in your feedback on this section in particular.  This sets up the example that we use for the rest of this doc (and in config-rss.toml).  Hopefully I got the details right!

Reply:
This looks good to me! This is a great primer for what the address configuration needs to look like. It makes me think that we should figure out a way to just put all this in one config file upfront to reduce the configuration complexity spread out over several different files.